frontend/ui/widget/touchmenu.lua

```diff
@@ -157,6 +157,9 @@ function TouchMenuItem:onTapSelect(arg, ges)
     end
     if enabled == false then return end
 
+    -- If the menu hasn't actually been drawn yet, don't do anything (as it's confusing, and the coordinates may be wrong).
```

Comment:
More importantly, it'll crash :D.

The "confusing" comment stems from my earlier experiment where I blindly called the actual callback in these cases.